expected-consensus.md

```diff
@@ -333,9 +333,17 @@ TODO: ensure 'power' is properly and clearly defined
 
 It is possible for forks to emerge naturally in Expected Consensus. EC relies on weighted chains in order to quickly converge on 'one true chain', with every block adding to the chain's weight. This means the heaviest chain should reflect the most amount of work performed, or in Filecoin's case, the most storage provided.
 
-The weight at each block is equal to its `ParentWeight`, plus that block's delta weight. Delta
-weight is a constant `V`, plus `X` - a function of the total power in the network as reported in the Power Table.  The exact value for `V` and the magnitude of the power ratio value are
-still to be determined, but for now `V = 10` and `X = log(TotalPower)`.
+The weight at each block is equal to its `ParentWeight` plus that block's delta weight. Delta
+weight is a term P times the sum of a constant `V` times the number of blocks included in that TipSet in the given round (can only be calculated in the next round), and `X` - a function of the total power in the network as reported in the Power Table. P will be equal to some term P_i^P_n and seeks to punish selfish mining. So we have:
+
+`Weight = ParentWeight + P(V*NumBlocksMinedInThisRound + X)`
+
+ The exact value for these parameters remain to be determined, but for now we set:
+
+- `V = 2 * number of blocks in the round`
+- `X = log(TotalPower in Power Table l blocks back)`
+- `P_i = .87`
```

Comment:
0.87 is an oddly specific number. Where does that come from?

Reply:
[The Weighting fn observable](https://observablehq.com/d/3812cd65c054082da).

Note 1: The number is most certainly going to change. Simulations shows it's not performing quite as I'd like yet. Regardless, should probably be removed from the spec I'd argue (in favor of our having a separate FIL-params guideline at Testnet that fills in variables).

Note 2: I'm cleaning this observable, the conf time one and the consensus repo this week, so I can make them all fully open and accessible to people farther away from the topic.